InstagramClone/InstagramClone/profilCell/ProfilStoryTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ProfilStoryTableViewCell.swift
+//  InstagramClone
+//
+//  Created by Николай Панов on 22.07.2021.
+//
+
+import UIKit
+
+final class ProfilStoryTableViewCell: UITableViewCell {
+    
+    // MARK: - IBOutlet
+    @IBOutlet weak var saveCollectionView: UICollectionView!
+    
+    //MARK: - Static Properties
+    static let identifier = "SaveStory"
+    
+    //MARK: - Static Methods
+    static func nib() -> UINib {
+        return UINib(nibName: "ProfilStoryTableViewCell", bundle: nil)
```

Comment:
```suggestion
        return UINib(nibName: "ProfilStoryTableViewCell", bundle: nil)
```
это же тоже id в константу его